SDK.ipynb

```diff
@@ -78,9 +91,21 @@
    },
    "outputs": [],
    "source": [
-    "import globus_sdk\n",
+    "# fill this line in with the code that you got\n",
+    "auth_code = \"YOUR CODE HERE\"\n",
```

Comment:
Not to be obnoxious, but it would be super nice if this were `"YOUR_CODE_HERE"` so I could double-click it to overwrite/paste.